kDriveTestShared/MCKTokenDelegate.swift

```diff
@@ -16,21 +16,17 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import XCTest
+import Foundation
+import InfomaniakCore
+import InfomaniakLogin
 
-/// Sanity checks of SWIFT_ACTIVE_COMPILATION_CONDITIONS
-final class TestsPreconditions: XCTestCase {
```

Comment:
This test was removed as the TEST flag never left the test code, not the app. This was a bug I introduced, will discuss it next Monday